android-components/components/ui/tabcounter/src/main/res/drawable/mozac_ic_mask.xml

```diff
@@ -0,0 +1,26 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape android:shape="oval">
+            <size
+                android:width="16dp"
+                android:height="16dp" />
+            <solid android:color="@color/mozac_ui_photonPurple60" />
+            <stroke android:color="@color/mozac_ui_icons_fill" android:width="1.5dp" />
+        </shape>
+    </item>
+    <item>
+        <vector
+            android:width="16dp"
+            android:height="16dp"
+            android:viewportHeight="20"
+            android:viewportWidth="20">
+            <path
+                android:fillColor="@color/mozac_ui_icons_fill"
+                android:fillType="evenOdd"
+                android:pathData="M12.49 6.99H13.77C14.44 6.99 14.98 7.53 14.98 8.2V10.2C14.98 11.21 14.48 12.15 13.64 12.71L13.48 12.82C13.28 12.95 13.05 13.02 12.81 13.02H12.47C12.22 13.02 11.98 12.94 11.77 12.79L10.64 11.98C10.49 11.87 10.3 11.81 10.12 11.81H9.84001C9.65001 11.81 9.47001 11.87 9.32001 11.98L8.19001 12.79C7.99001 12.94 7.74001 13.02 7.49001 13.02H7.15001C6.91001 13.02 6.68001 12.95 6.48001 12.82L6.32001 12.71C5.48001 12.15 4.98001 11.21 4.98001 10.2V8.2C4.98001 7.53 5.52001 6.99 6.19001 6.99H7.49001C8.20001 6.99 8.86001 7.31 9.30001 7.86C9.47001 8.07 9.72001 8.19 9.99001 8.19C10.26 8.19 10.51 8.07 10.68 7.86C11.12 7.31 11.79 6.99 12.49 6.99ZM8.87 10.19C8.55 10.55 8.05 10.78 7.5 10.78C6.95 10.78 6.45 10.55 6.13 10.19C5.96 10 5.96 9.7 6.13 9.51C6.45 9.15 6.95 8.92 7.5 8.92C8.05 8.92 8.55 9.15 8.87 9.51C9.04 9.7 9.04 10 8.87 10.19ZM11.13 10.19C11.45 10.55 11.95 10.78 12.5 10.78C13.05 10.78 13.55 10.55 13.87 10.19C14.04 10 14.04 9.7 13.87 9.51C13.55 9.15 13.05 8.92 12.5 8.92C11.95 8.92 11.45 9.15 11.13 9.51C10.96 9.7 10.96 10 11.13 10.19Z" />
+        </vector>
```

Comment:
I was inspecting the main component of the [icon in the figma](https://www.figma.com/file/Ha2T4I0nX0h7B95GYxH6wV/v1-ABM-minimal-UI?type=design&node-id=2585-14916&mode=design&t=UMVacF6H6bTpiV7e-0) which lead me to [icon in the design system](https://www.figma.com/file/Kv7wWr8vH6AQrGhu9vmjdE/Mobile-Icons?type=design&node-id=2129-1788&mode=design&t=uQN7TScb15AEnExX-0) - Right Click on the icon -> Select "Go to main component". 

Could we instead turn this into an `<item android:drawable>` and add [mozac_ic_private_mode_circle_fill_20](https://github.com/FirefoxUX/acorn-icons/blob/main/icons/mobile/20/xml/ic_private_mode_circle_fill_20.xml) to ui-icons so that it could be used here?

This also part of my larger initiative of maintaining a single source of truth of our icons and aligning it with [icon design system](https://github.com/FirefoxUX/acorn-icons/tree/main). I am in the process of importing and aligning our icons with this design system in https://github.com/mozilla-mobile/firefox-android/pull/3017 and you will find even more details in [Bug 1842389](https://bugzilla.mozilla.org/show_bug.cgi?id=1842389). This will help us deduplicate icons that exist in both Fenix and AC, and allow us to always be using the latest icons that comes out from UX.